aggregator/src/aggregator.rs

```diff
@@ -3164,24 +3308,31 @@ async fn send_request_to_helper<T: Encode>(
     method: Method,
     url: Url,
     route_label: &'static str,
-    content_type: &str,
-    request: T,
+    request_body: Option<RequestBody<T>>,
```

Comment:
The `DELETE` request to aggregation jobs has no body, so I needed to make those parameters optional. I combined them into `struct RequestBody` so that (1) we could have a single `Option<>` in the argument list instead of separate `Option<&str>` and `Option<T>` which need to be `Some` or `None` in lockstep and (2) used a struct instead of a tuple for readability.